src/transformers/utils/dummy_tokenizers_objects.py

```diff
@@ -38,6 +38,15 @@ def from_pretrained(self, *args, **kwargs):
         requires_backends(self, ["tokenizers"])
 
 
+class BigBirdPegasusTokenizerFast:
```

Comment:
@patrickvonplaten if I am not wrong, I need to remove this right??

Reply:
Yes, actually it should be removed automatically when running `make fix-copies` I think :-)

Reply:
When I am running `make fix-copies`, its not removing it rather putting it back. Please help me in solving this.